defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+wp_path:      "/usr/local/bin/wp" 
```

Comment:
1 space between directive / value - this applies to the entire PR.  The meta/main.yml looks good.

Reply:
Fixed spacing across the board.